website/docs/getting-started/setup/installation-guides/README.md

```diff
@@ -90,14 +90,6 @@
              </a>     
         </div> 
         <b><a href="/getting-started/setup/installation-guides/restack">Restack</a></b>
-    </div>
-    <div class="columnGrid column-two" align="center">
-        <div class="containerCol">
-            <a href="/getting-started/setup/installation-guides/ansible">
-            <img class="containerImage" src="/img/Ansible-logo.png" alt="Ansible-logo"/>
-            </a>     
-        </div> 
-         <b><a href="/getting-started/setup/installation-guides/ansible">Ansible</a></b>
     </div>
      <div class="columnGrid column-three" align="center">
```

Comment:
```suggestion
     <div class="columnGrid column-two" align="center">
```

Reply:
<img width="961" alt="Screenshot 2023-02-13 at 13 48 01" src="https://user-images.githubusercontent.com/32227135/218405936-45f27f3a-583f-4913-a0c5-7fbf422c1ed5.png">

Also, keep column three empty to make it all look aligned